docs/swarm/swarm-tutorial/create-swarm.md

```diff
@@ -25,6 +25,8 @@ node. For example, the tutorial uses a machine named `manager1`.
     docker swarm init --advertise-addr <MANAGER-IP>
     ```
 
+  >**Note:** If you are using Docker for Mac or Docker for Windows to test single-node swarm, simply run `docker swarm init` with no arguments. There is no need to advertise an IP address in this case. This will initiate a swarm with a manager as the single node in the swarm. You can create and scale services on a single-node swarm. You can also create a multi-node swarm with Docker Machine. To learn more, see the setup topic on [Using Docker for Mac or Docker for Windows](index.md#using-docker-for-mac-or-docker-for-windows) with Swarm mode.
+
```

Comment:
I think it is better to say "there is no need to specify a `--advertise-addr` in this case" as there is actually one set, it is just not one you can connect to externally at present.